### DISCUSSION TITLE
about mhm installation

### DISCUSSION THREAD

**Discussion by manisasah** [mhm-ufz/mHM#12] [Installation] [answered]
Hello
when ever i type make and cmake command in cygwin terminal its says bad command or command not found.in tutorial video their mhm folder has make inside but its not present in mine. How can i resolve this issue?

**Comment by MuellerSeb** [comment]
Hello there,
sorry to hear that you got problems installing mHM. Did you take a look at the create video guide of @cuneyd on Youtube:
https://www.youtube.com/watch?v=FGJOcYEzbP4

Hope that helps. ATM I don't have a possibility to check it myself, but I will come back to this later.
Cheers, Sebastian

**Comment by cuneyd** [comment, marked as answer]
Hello Manisasah,

Apparently you didn’t install the libs from setup_64.exe in cygwin. Please follow the steps in the video and make sure that you don’t get those warnings saying “bad command”.

**Comment by manisasah** [comment]
Thanx for your suggestion



*with best regards,*
*Manisha sah,*
*MSc. Interdisciplinary Water Resource Management*
*Nepal Engineering College*
